.eslintrc.js

```diff
@@ -241,7 +241,7 @@ module.exports = {
 			rules: {
 				'@typescript-eslint/no-explicit-any': 'error',
 				'no-use-before-define': 'off',
-				'@typescript-eslint/no-use-before-define': [ 'error' ],
+				'@typescript-eslint/no-use-before-define': 'off',
```

Comment:
This is needed because in the new Interactivity Store API we need to access the `state`,  `actions` and other components from the store before the store is defined, as you can see here: https://github.com/woocommerce/woocommerce-blocks/pull/11721/files#diff-40099b9b90dd057ec2594e0ceee370c105bbf64c58b94396e278b2a7819b0abdR155

Basically, we destructure the return of the `store` call and use the `state` before that declaration in here:
https://github.com/woocommerce/woocommerce-blocks/pull/11721/files#diff-40099b9b90dd057ec2594e0ceee370c105bbf64c58b94396e278b2a7819b0abdR51

Reply:
Solved by moving the store to the top and https://github.com/woocommerce/woocommerce-blocks/pull/11721#issuecomment-1808961451.